db/articles.js

```diff
@@ -0,0 +1,74 @@
+(function () {
+  const articles = {
+    "articles": [
+      {
+        title: "My Swedish-made Penis Enlarger and Me",
+        author: "Austin Powers",
+        body: "That's not mine",
+      }
+    ]
+  };
+
+  const getArticles = () => {
+    return articles;
+  };
+
+  const addArticle = (title, body, author) => {
+    let article = {};
+
+    article.title = title;
+    article.body = body;
+    article.author = author;
+    articles.articles.push(article);
+  };
+
+  const findArticle = (title) => {
+    let index = -1;
+    let formattedArg = title.split(' ').join('').toLowerCase();
+
+    articles.articles.forEach(article => {
+      if (article.title.split(' ').join('').toLowerCase() === formattedArg) {
+        return index = articles.articles.indexOf(article);
+      }
+    });
+
+    if (index < 0) {
+      return false;
```

Comment:
You could have started with the index = false and not needed this section of code.